plugins/modules/purefa_pgsnap.py

```diff
@@ -144,7 +144,6 @@
     restore: data
     target: data2
     overwrite: true
-    with_default_protection: false
```

Comment:
Default value for **with_default_protection** is set to true, so if user don't give this option, it will fail with error - "overwrite and add_to_pgs or with_default_protection are incompatable". Better to retain the example so user will set it to false or we can set default value to false.